### ISSUE TITLE
Bug Report: Weekend 1 Erect shader boundaries are cut off abruptly

### ISSUE THREAD

**Issue by Average-FNF-Modder** [FunkinCrew/Funkin#3248]
### Issue Checklist

- [X] I have properly named the issue
- [X] I have checked the issues/discussions pages to see if the issue has been previously reported

### What platform are you using?

Itch.io (Downloadable Build) - Windows

### If you are playing on a browser, which one are you using?

None

### Version

0.5.0

### Context (Provide images, videos, etc.)

I have a chart that zooms out pretty far on this stage, but unfortunately when I switched to the erect version in the new update for some mood the effect couldn't cover all my zoom. I'm not sure if this _fully_ counts as a _bug_ persay, but it certainly is annoying and makes using the base game stages a little worse.

### Steps to reproduce (or crash logs, errors, etc.)

Showcase of it:
![image](https://github.com/user-attachments/assets/21cca7f7-f433-493a-bed0-8f93715a3e2e)

Right at the clipping point:
![image](https://github.com/user-attachments/assets/f477ca6b-c929-48e3-b1a4-ed4dc1574b6b)

Just zoom out decently far on one of the charts (This probably also works on the other erect stages with mood lighting if I had to guess).

**Comment by Average-FNF-Modder** [comment]
I have also recently noticed that this causes real issues outside of the editor because when the Darnell (BF Mix) ends the screens scrolls up and I do notice the clipping and it looks odd. Like it's fine if you don't notice but once you see it once it'll haunt you every time you beat the song.

**Comment by Hundrec** [comment]
I wonder if expanding the shader effect impacts performance, even if it's just by a tiny bit.
Though I notice this every time I play Darnell (BF Mix), so I'd prefer that the shader be exactly the size of the stage.

**Comment by Average-FNF-Modder** [comment]
> I wonder if expanding the shader effect impacts performance, even if it's just by a tiny bit. Though I notice this every time I play Darnell (BF Mix), so I'd prefer that the shader be exactly the size of the stage.

Yeah, same. Like I don't expect it to go on forever, but at least as far as you can zoom out before leaving stage boundaries.
